src/mmw/sass/components/_modals.scss

```diff
@@ -138,3 +138,20 @@
   top: 1rem;
   left: 2rem;
 }
+
+.fa {
+  font-size: 1.0rem;
+  padding: 0 0.1rem 0 0.1rem;
+}
```

Comment:
Seems to the analyze tab while that's running, too:

![screen shot 2016-10-05 at 2 41 21 pm](https://cloud.githubusercontent.com/assets/4165523/19126672/d3575724-8b09-11e6-92b9-07171303261c.png)